keyboards/atreus/keymaps/khitsule/keymap.c

```diff
@@ -3,8 +3,10 @@
 #include QMK_KEYBOARD_H
 
 #define ALPH 0
-#define LOWR 1
-#define RAIS 2
+#define GAME 1
+#define LOWR 2
+#define RAIS 3
+#define ADJS 16
```

Comment:
This bloats firmware size with empty layers, however you might want to swap to an enum

```c
enum my_layers {
  ALPH,
  GAME,
  LOWR,
  RAIS,
  ADJS,
};
```

```suggestion
#define ADJS 4
```

Reply:
Old habits die hard, Ill start doing that in future keymap updates/refinements. Thanks